lib/phoenix/token.ex

```diff
@@ -109,6 +119,7 @@ defmodule Phoenix.Token do
       86400 seconds (1 day) and it may be overridden on verify/4.
```

Comment:
```suggestion
      86400 seconds (1 day) and it may be overridden on `Plug.Crypto.verify/4`.
```

That fn belongs to another lib/context so IMO it's worth adding a proper link as it's done on other parts of this doc.